.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+
+on:
+  pull_request:
+    branches: ['*']
+  push:
+    branches: ['master']
+    tags: ['v*']
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        java: ['adopt@1.8', 'adopt@1.11']
+        scala: ['2.11.12', '2.12.11', '2.13.3', '0.27.0-RC1']
+    steps:
+    - uses: actions/checkout@v2.3.4
+    - uses: olafurpg/setup-scala@v10
+      with:
+        java-version: ${{ matrix.java }}
+    - name: Cache scala dependencies
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/.ivy2/cache
+          ~/.sbt
+          ~/.m2
+          ~/.cache
+        key: sbt-nio-cache-${{ matrix.java }}-${{ matrix.scala }}
```

Comment:
I think we don't even need `nio` prefix. As far as I understood it's project-specific, or?

Also, I've used a bit different naming scheme, see [here](https://github.com/zio/zio-redis/blob/master/.github/workflows/ci.yml#L30).